team/index.html

```diff
@@ -1472,6 +1472,22 @@ <h1>Developers</h1>
 						<span>Android Developer</span>
 					</div>
 				</div>
+				<div class="col-md-3 col-sm-6 speaker-column">
+					<div class="speaker">
+						<div class="image-holder">
+							<img class="background-image" alt="Amrtya Mondal" src="../img/team/amrtyamondal.png">
+							<div class="hover-state text-center preserve3d">
+								<div class="social-links vertical-align">
+									<a href="<a href="https://github.com/neel1998" target="blank"><i class="icon fa fa-github"></i></a>" target="blank"><i class="icon fa fa-github"></i></a>
```

Comment:
Wrong GitHub link. Please change it. The correct link should be ```https://github.com/atm1504```.
Wrong spelling in alt tag